lib/lightning_web/live/run_live/show.ex

```diff
@@ -27,6 +57,12 @@ defmodule LightningWeb.RunLive.Show do
             <span class="pl-2 font-light">
               <%= display_short_uuid(@id) %>
             </span>
+            <div class="mx-2"></div>
+            <.async_result :let={run} assign={@run}>
+              <%= if run do %>
+                <.snapshot_version run={run} workflow={@workflow} />
+              <% end %>
+            </.async_result>
```

Comment:
fyi @elias-ba I rewrote this to use a component, please avoid assigning variables inside heex templates. I also changed `JobView.job_edit_view` and moved the editor disabled data into assigns.